src/DssSpell.t.sol

```diff
@@ -1111,4 +1120,52 @@ contract DssSpellTest is DSTest, DSMath {
         assertEq(MedianAbstract(UNIUSD_MED).bud(SET_UNI), 1);
     }
 
+    event Log(string message, address deployer, bytes32 contractName);
+
+    function hasWards(address addr) internal returns (bool) {
+        (bool ok, bytes memory data) = addr
+            .call(abi.encodeWithSignature("wards(address)", 0x0));
+        return ok && data.length == 32;
+    }
+
+    address[] deployerAddresses = [
+        0xdDb108893104dE4E1C6d0E47c42237dB4E617ACc,
+        0xDa0FaB05039809e63C5D068c897c3e602fA97457,
+        0xda0fab060e6cc7b1C0AA105d29Bd50D71f036711,
+        0xDA0FaB0700A4389F6E6679aBAb1692B4601ce9bf,
+        0x0048d6225D1F3eA4385627eFDC5B4709Cab4A21c
+    ];
+
+    function hasBadAuth(address addr, bytes32 contractName) internal {
+        for (uint i = 0; i < deployerAddresses.length; i ++) {
+            if (AuthLike(addr).wards(deployerAddresses[i]) > 0) {
```

Comment:
This is probably fine. All of our Auth checks should fail if `ward != 1`, but it would be interesting to catch something unexpected here.